cli/azd/pkg/environment/environment.go

```diff
@@ -177,6 +181,7 @@ func (e *Environment) LookupEnv(key string) (string, bool) {
 // does not exist. [Save] should be called to ensure this change is persisted.
 func (e *Environment) DotenvDelete(key string) {
```

Comment:
Can we rename these `DotXXX` methods?  I know we use the `dotenv` package behind the scenes but as a consumer of the `Environment` struct we don't care or need to know.  I would prefer to see something more generic like `DeleteValue(...)` and `SetValue(...)`

Reply:
I do like these new names better, I will do this in a follow up PR since it will touch more stuff.